src/claimable_balances/single_claimable_balance_request.rs

```diff
@@ -1,30 +1,43 @@
 use crate::models::*;
 
+
+#[derive(Default, Clone)]
+pub struct ClaimableBalanceId(String);
+#[derive(Default, Clone)]
+pub struct NoClaimableBalanceId;
+
```

Comment:
We should add some documentation on the context of each of these states. What do they mean?

Reply:
Moved to https://github.com/Baseflow/stellar-rust-sdk/issues/4